### ISSUE TITLE
Raphael Hayruni Review

### ISSUE THREAD

**Issue by R4phae1** [GrigTukhar/ETPG_OOP#1]
first of all, I would like to start stating that the amount of work that Griqor did is outstanding. it's not only the amount but the quality as well. I find this project as a very useful and creative one. All in all, nothing is perfect. Though it is obvious that my programming knowledge is not letting me judge the work of Grig's but I think I managed to find some cons in this code. 

1. It is so hard to read and understand. The written complexity is very high, I think there is some room for improvement in that area. 

2. There is no verification for the birthdate. you can write any name on a birth date and the program will take the information as a valid one. 

3. there is no a coach class in the code, he just added everything in the class called Athlete.

To sum up, I just want to congratulate Griqor on a great job he did.
